scripts/ticketmaster-API.py

```diff
@@ -15,35 +21,49 @@
 )
 
 cursor = db.cursor()
+
+### save API URLs as vars
 URL = "https://app.ticketmaster.com/discovery/v2/events.json"
```

Comment:
these are not very descriptive. it's better to call them something like `ticketmaster_url`